benchmarks/runners/apfelpy_.py

```diff
@@ -88,7 +84,7 @@ def benchmark_polarized(self, pto):
         self.run(
             self.theory_updates_zm(pto),
             self.obs_updates_pol(),
-            ["ToyLH"],
+            ["ToyLH_polarized"],
```

Comment:
Using `ToyLH_polarized` significantly worsen the agreement; at LO the difference can be up to 12%. This also happens when benchmarking against Apfel.